### ISSUE TITLE
Pagination of epub book

### ISSUE THREAD

**Issue by endifai** [futurepress/epub.js#1144]
How can i count all pages by fixed layout instead of count of chars? I saw that in epub js there is a feature to get an actual pages for chapters but not for all book.

**Comment by Aryesh-M** [comment]
Yes, I am facing the same issue. Please guide how that can be possible without loading each section at a time. @fchasen
